test_suite/BUILD

```diff
@@ -0,0 +1,18 @@
+[
+    py_test(
+        name = "starlark_test_" + test_file.replace(".", "_"),
+        srcs = [
+            "starlark_test.py",
+            "testenv.py",
+        ],
+        args = [test_file],
+        data = [
+            "@io_bazel//src/main/java/com/google/devtools/starlark:Starlark",
+            "@net_starlark_go//cmd/starlark:starlark",
+            "@starlark-rust//:starlark",
```

Comment:
Maybe separate the interpreters. It can be useful to test only one interpreter.
I'd copy the list comprehension 3 times, once for each interpreter to test.